crates/compiler/Cargo.toml

```diff
@@ -15,6 +15,12 @@ rust-version.workspace = true
 
 [dependencies]
 bimap.workspace = true
+cairo-lang-compiler = { path = "../../cairo/crates/cairo-lang-compiler" }
+cairo-lang-defs = { path = "../../cairo/crates/cairo-lang-defs" }
+cairo-lang-filesystem = { path = "../../cairo/crates/cairo-lang-filesystem" }
+cairo-lang-lowering = { path = "../../cairo/crates/cairo-lang-lowering" }
+cairo-lang-semantic = { path = "../../cairo/crates/cairo-lang-semantic" }
+cairo-lang-utils = { path = "../../cairo/crates/cairo-lang-utils" }
```

Comment:
Why are these introduced to the compiler? We don't currently depend on any of them. Please remove them.